src/EnergyPlus/DataHeatBalSurface.hh

```diff
@@ -110,6 +110,7 @@ struct HeatBalSurfData : BaseGlobalStruct
     Array1D<Real64> SurfTempInsOld;      // SurfTempIn from previous iteration for convergence check
     Array1D<Real64> SurfTempInTmp;       // Inside Surface Temperature Of Each Heat Transfer Surface
     Array1D<Real64> SurfHcExt;           // Outside Convection Coefficient
+    Array1D<Real64> CoeffAdjRatioOut;    // Outside Convective and Radiative Coefficient Adjustment Ratio
```

Comment:
Rename as `SurfWinCoeffAdjRatioOut`. Add the comment that only for exterior windows.

Reply:
Thanks. Corrected it.